Classes/Form/Field/Text.php

```diff
@@ -55,12 +61,9 @@ public function buildConfiguration()
         $configuration['rows'] = $this->getRows();
         $configuration['cols'] = $this->getColumns();
         $configuration['eval'] = $this->getValidate();
-        $defaultExtras = $this->getDefaultExtras();
-        if (true === $this->getEnableRichText() && true === empty($defaultExtras)) {
-            $typoScript = $this->getConfigurationService()->getAllTypoScript();
-            $configuration['defaultExtras'] = $typoScript['plugin']['tx_flux']['settings']['flexform']['rteDefaults'];
-        } else {
-            $configuration['defaultExtras'] = $defaultExtras;
+        if (true === $this->getEnableRichText()) {
```

Comment:
We need a condition here which checks if either ckeditor or rtehtmlarea is installed and does the appropriate thing depending on that. We cannot yet remove the existing behavior, but it is perfectly fine to only trigger it if rtehtmlarea is installed.

The current TS usage can be marked deprecated, iirc it's added in ext_localconf.php.